### DISCUSSION TITLE
I can not sign out of my Github account

### DISCUSSION THREAD

**Discussion by raulfdeztdo** [zed-industries/zed#7693] [Feature Requests]
The problem, is that I made the "Sign in" with a Github account that I don't want, when I sign out it doesn't open the github page again, it logs in the same as before, I have revoked the access from the unwanted github account even so, it continues to login on the same erroneous account.

Any solution?

**Comment by mikayla-maki** [comment]
We haven't implemented log out yet 😅, you can delete the credentials from your macOS keychain though!
